src/SDKs/CognitiveServices/dataPlane/Search/Search/Properties/AssemblyInfo.cs

```diff
@@ -1,19 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System.Reflection;
```

Comment:
Why is assemblyInfo is being deleted?

Reply:
Because it was creating a build error

Reply:
Duplicate assembly information yada yada, it's because the same information is coming from the nuget fields of the csproj (As far as I gathered from Google :P)

Reply:
@stefanvalianu if you pull in the latest from upstream, you should not see this error. Recently due to upgrade in some references, this error started showing up. Once you pull in the changes you should not see this error anymore.

Reply:
Gotcha, lemme try that